src/test/java/org/wof/foodministry/FoodEntryTestClass.java

```diff
@@ -43,13 +43,48 @@ public void testGetWeightFalse() {
         int result = entry.getWeight();
```

Comment:
I would turn this into it's own test case ie `public void testGetWeightFalse()`
Don't for get to add the `@Test` above the method